amlb/utils/core.py

```diff
@@ -148,7 +163,7 @@ def __add__(self, other):
             self.__dict__.update(other)
         return self
 
-    def __mod__(self, other):
+    def __or__(self, other):
```

Comment:
I appreciate this change. Is there a specific reason you did not change it to `__ior__`?